CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.11.0 / 2023-12-08
```

Comment:
@SuperQ reminder to change the date before merge

Reply:
I'm not too strict on these dates.